src/pdffontdataopentype.cpp

```diff
@@ -621,7 +621,7 @@ wxPdfFontDataOpenTypeUnicode::WriteUnicodeMap(wxOutputStream* mapData,
 size_t
 wxPdfFontDataOpenTypeUnicode::WriteCIDSet(wxOutputStream* setData,
   const wxPdfEncoding* encoding,
-  wxPdfSortedArrayInt* usedGlyphs,
+  wxPdfSortedArrayInt* /* usedGlyphs */,
```

Comment:
For the parameters `encoding` and `subsetGlyphs` that are also unused in this method I used
```C++
  wxUnusedVar(encoding);
  wxUnusedVar(subsetGlyphs);
```
to mark them as unused in the method body. IMHO we should be consistent by either adding `wxUnusedVar(usedGlyphs)` in the method body or by removing the parameter names for all unused parameters. I'm not sure what would be the preferred approach.

Reply:
Oops, sorry, I didn't notice this somehow. I should have definitely just added `wxUnusedVar(usedGlyphs)` below. Should I do this or will you when merging?

Reply:
Please be so kind to take care of this change. TIA.